LeetCode/1340_Jump_Game_V.py

```diff
@@ -0,0 +1,28 @@
+class Solution(object):
+    def maxJumps(self, arr, d):
+        """
+        :type arr: List[int]
+        :type d: int
+        :rtype: int
+        """
+
+        dp = [1] * len(arr)
+
+        for i, _ in sorted(enumerate(arr), key=lambda x: x[1]):
+            max_h = None
```

Comment:
Code does not pass Test on LeetCode

![image](https://user-images.githubusercontent.com/6342890/95656648-0834ea80-0b10-11eb-8cd3-c5e6ce47b586.png)